src/database/seeds/0002-update-canonical_fields.sql

```diff
@@ -0,0 +1,188 @@
+('Proposal Funds Locations Under US Sanction Explanation', 'proposal_funds_locations_under_us_sanction_explanation', '{ "type": "string" }' ),
+('Proposal Supports Research', 'proposal_supports_research', '{ "type": "boolean" }' ),
+('Proposal Involves Desk Research', 'proposal_involves_desk_research', '{ "type": "boolean" }' ),
+('Proposal Collects New Data from Individuals', 'proposal_collects_new_data_from_individuals', '{ "type": "boolean" }' ),
+('Proposal Collects Personal Data', 'proposal_collects_personal_data', '{ "type": "boolean" }' ),
+('Proposal Collects New or Personal Data Explanation', 'proposal_collects_new_or_personal_data_explanation', '{ "type": "string" }' ),
+('Proposal Research Will be Published', 'proposal_research_will_be_published', '{ "type": "boolean" }' ),
+('Organization has Human Subjects Research Policy', 'organization_has_human_subjects_research_policy', '{ "type": "boolean" }' ),
+('Proposal Research Involves Minors', 'proposal_research_involves_minors', '{ "type": "boolean" }' ),
+('Organization has Child Protection Policy', 'organization_has_child_protection_policy', '{ "type": "boolean" }' ),
+('Proposal Funds Fiscal Agent Duties Research or Conference', 'proposal_funds_fiscal_agent_duties_research_or_conference', '{ "type": "boolean" }' ),
+('Proposal Funds Fiscal Agent Duties Research or Conference Explanation', 'proposal_funds_fiscal_agent_duties_research_or_conference_explanation', '{ "type": "string" }' ),
+('Additional Information From Applicant', 'additional_information_from_applicant', '{ "type": "string" }' ),
+('Organization Articles of Incorporation Document', 'organization_articles_of_incorporation_document', '{ "type": "uri" }' ),
+('Organization Child Protection Policy Document', 'organization_child_protection_policy_document', '{ "type": "uri" }' ),
+('Organization Conflict of Interest Policy Document', 'organization_conflict_of_interest_policy_document', '{ "type": "uri" }' ),
+('Organization Human Subjects Research Policy Document', 'organization_human_subjects_research_policy_document', '{ "type": "uri" }' ),
+('Organization IRS Tax Letter Document', 'organization_irs_tax_letter_document', '{ "type": "uri" }' ),
+('Organization Latest Unaudited Financial Statement Document', 'organization_latest_unaudited_financial_statement_document', '{ "type": "uri" }' ),
+('Organization Charter Document', 'organization_charter_document', '{ "type": "uri" }' ),
+('Organization Policy Regarding Data Gathering from Children Document', 'organization_policy_regarding_data_gathering_from_children_document', '{ "type": "uri" }' ),
+('Organization Whistleblower Policy Document', 'organization_whistleblower_policy_document', '{ "type": "uri" }' ),
+('Organization Latitude EPSG:4326', 'organization_latitude_epsg_4326', '{ "type": "number" }' ),
+('Organization Longitude EPSG:4326', 'organization_longitude_epsg_4326', '{ "type": "number" }' ),
```

Comment:
This feels like another example of a split data type: having one without the other is useless, and mixing values from one version to another is actively incorrect. This is actually a useful place for a URI! Specifically, a [`geo` URI](https://en.wikipedia.org/wiki/Geo_URI_scheme).

Reply:
Good call, yes, a geo URI would be better here.

Reply:
Again, I agree that in the long run we want to collapse some data fields and shore them up, that will need to be done with our new approach(es) to fields. This change happened in the pilot production and is still more useful merged than unmerged.

Reply:
#603 might be the right place to consider this too.